control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp

```diff
@@ -185,9 +185,6 @@ class TRAJECTORY_FOLLOWER_PUBLIC PidLongitudinalController : public Longitudinal
   double m_ego_nearest_dist_threshold;
   double m_ego_nearest_yaw_threshold;
 
-  // 1st order lowpass filter for acceleration
-  std::shared_ptr<trajectory_follower::LowpassFilter1d> m_lpf_acc{nullptr};
-
```

Comment:
Do you think it's ok to remove the filter for acceleration? I guess whether the acceleration from the localization module may not be smooth enough for the controller.

Reply:
As long as it is OK for planning modules to use acceleration with unified smoothing parameter defined in localization/twist2accel, I guess planning modules do not need to filter acceleration by their own (Not sure if this assumption is correct or not, though).

Reply:
:sorehasou:

Reply:
@kminoda 
FYI.
As you may know, applying lowpass filter has pros/cons.
- pros: the value will be smooth.
- cons: the value will have a delay

How much planning/control modules wanna apply lowpass filter depends on how they wanna use the filtered value.
Therefore, in general, the filter gain should be tuned for each use-cases in each module.

Reply:
Minoda-san investigated the LPF parameters in https://github.com/autowarefoundation/autoware.universe/pull/1089. I think taking a default LPF gain causes no problem.

Reply:
Got it. thanks.